apps/server/src/shared/domain/mapper/current-user.mapper.ts

```diff
@@ -0,0 +1,34 @@
+import { Role, User } from '../entity';
+import { ICurrentUser } from '../interface';
+
+export class CurrentUserMapper {
```

Comment:
For now it is only used in authentication module. Please hold it on this place. We want to reduce shared folder in future and encapsulate more code in the modules.

Reply:
We placed the mapper close to the `ICurrentUser` interface, because they are related to each other. If we place the mapper in the authentication module we should also move the interface. Otherwise the relationship between these two is not clear. Since `ICurrentUser` is used by nearly all modules it should be palced in shared.

Reply:
@Metauriel what is your opinion?

Reply:
@Metauriel problem is solved. ICurrentUser is moved in a additional ticket to authentication modul.